src/cpp/session/modules/SessionConnections.R

```diff
@@ -160,26 +160,107 @@ options(connectionObserver = list(
    NULL
 })
 
+.rs.addFunction("connectionReadSnippets", function() {
+   snippetsPath <- getOption("connections-path", "/etc/rstudio/connections/")
```

Comment:
@jmcphers that's what we already have in this PR, correct? Just checking I'm not missing any change form this comment.